tests/testthat/_snaps/class.md

```diff
@@ -4,71 +4,74 @@
       bake_check_class_core(x1, "character", "x1")
     Condition
       Error in `bake_check_class_core()`:
-      ! x1 should have the class(es) character but has the class(es) numeric.
+      ! `x1` should have the class "character" but only has the class "numeric".
```

Comment:
```suggestion
      ! `x1` should have the class "character" but has the class "numeric".
```

I'd say no need to add this "only."

Reply:
done in https://github.com/tidymodels/recipes/pull/1255/commits/483af8a84507657d909bec152f8bb87e15fb9350